### DISCUSSION TITLE
Wireguard and ROOT

### DISCUSSION THREAD

**Discussion by lion1313** [tteck/Proxmox#2691] [Help Needed] [answered]
Hello, I am new to the proxmox world and I found your work great!!!! thanks again !
I installed WIREGUARD and noticed that there was an autoconnection, but I don't have the ROOT password (when I want to connect by SSH from another machine to the LXC)
What is the password? how is it defined during installation? can I change it?
Thanks again :)

**Comment by tteck** [comment, marked as answer]
When creating an application LXC, you are presented with two options, default settings or advanced settings. Default settings utilizes autologin (no root password) which means no SSH access. If SSH access is required, one needs to select Advanced during the LXC creation which will allow a root password to be set and an option to enable SSH

**Reply by perdakovich** [reply]
How can I update these settings if I installed LXC with default settings?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/324

**Reply by perdakovich** [reply]
Thanks!
